CONTRIBUTING.md

```diff
@@ -18,20 +18,28 @@ Articles should cover any topic related to practical, day-to-day usage of these
 
 - Submit articles with a PR on the [theheadless.dev repo](https://github.com/checkly/theheadless.dev)
 - Write articles in markdown and can use the [Markdown extensions provided by Vue press](https://vuepress.vuejs.org/guide/markdown.html)
-- Add articles to the `posts` directory as markdown files 
-- Add code examples to the `snippet` directory as Javascript files and inserted into markdown code blocks using the `<< @snippet` syntax 
+- Add articles to the `posts` directory as markdown files
+- Add code examples to the `snippet` directory as Javascript files and inserted into markdown code blocks using the `<< @snippet` syntax
 - Code examples should cover both Puppeteer and Playwright where applicable
 - Add your full name and GitHub username in the relevant `author` and `githubUser` front matter
 
 You can find additional styling guidelines in this [dedicated file](https://github.com/checkly/theheadless.dev/blob/master/STYLE.MD). Our team will gladly help you apply these after the initial review.
 
+### How to contribute
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) this repository to your own GitHub account and then [clone](https://help.github.com/articles/cloning-a-repository/) it to your local device.
+2. Create a new branch `git checkout -b MY_BRANCH_NAME`
+3. Install the dependencies: `npm install`
+5. Run `npm run dev` to build and watch for code changes
+6. The development branch is `canary` (this is the branch pull requests should be made against). On a release, the relevant parts of the changes in the `canary` branch are rebased into `master`.
```

Comment:
So far this hadn't been the workflow, is this a proposal to adopt it?

Reply:
@ragog what is the current workflow? I care most about having it described not so much about the workflow itself.

Reply:
@hlenke right now we do not have a development branch; I see this as becoming useful later on if we have more frequent contributions, but for now it seems enough to just have pull requests to master.